### PR TITLE
fozzie-components@7.41.0 - Use latest eslint-config-fozzie

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Changed
- Use latest eslint-config-fozzie.

### CODE REVIEW

**Review by ashleynolan — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

**Comment by xander-marjoram** [comment]
Will wait until #2208 is merged which may resolve/simplify some of the linting issues

File: package.json
Comment (xander-marjoram) [inline package.json:120]: Any higher than this seems to trigger the threat detection on my machine, I'm raising this with infosec.

**Review by kevinrodrigues — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**
